hw/formal/tools/jaspergold/fpv.tcl

```diff
@@ -67,13 +67,15 @@ if {$stopat ne ""} {
 }
 
 if {[info exists ::env(AFTER_LOAD)]} {
-    if {$env(AFTER_LOAD) != ""} {
-        puts "Running prefix TCL command from $env(AFTER_LOAD)"
-        source $env(AFTER_LOAD)
+    set flist $env(AFTER_LOAD)
+    foreach file $flist {
+        if {$file != ""} {
+            puts "Running prefix TCL command from $env(AFTER_LOAD)"
```

Comment:
I think this probably needs to be `$file`?

Reply:
I agree. Resolved